include/freerdp/server/audin.h

```diff
@@ -38,89 +39,141 @@ extern "C"
 
 	typedef struct s_audin_server_context audin_server_context;
 
-	typedef BOOL (*psAudinServerChannelIdAssigned)(audin_server_context* context, UINT32 channelId);
+	typedef BOOL (*psAudinServerChannelOpen)(audin_server_context* context);
+	typedef BOOL (*psAudinServerChannelIsOpen)(audin_server_context* context);
```

Comment:
Shall we name it `psAudinServerChannelIsOpened` instead ?

Reply:
hm..I was thinking about that name too:

`Close()` closes the DVC and shut downs the channel thread handling.
`Open()` sets up the channel thread handling and opens the DVC by asking the client side to create the respective DVC.

But `IsOpen()` just mentions, that the channel thread handling is set up. To actually know, that the DVC is open, it needs to wait for the client response of the DVC creation request.
But `IsOpen()` can only be `TRUE`, in the time period between `Open()` and `Close()`.